include/znc/IRCSock.h

```diff
@@ -150,6 +150,7 @@ class CIRCSock : public CIRCSocket {
     bool HasUHNames() const { return m_bUHNames; }
     bool HasAwayNotify() const { return m_bAwayNotify; }
     bool HasAccountNotify() const { return m_bAccountNotify; }
+    bool HasAccountTag() const { return m_bAccountTag; }
```

Comment:
Unused? I don't think it worth making all caps to have such accessors, if there is a way to query cap support by name.